spk/tt-rss/src/bin/php-settings.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+function executable_for_module ()
+{
+	echo -n "$*" | tr '[:upper:]' '[:lower:]' | tr -d '.'
+}
+
+function guess_php_installation () 
+{
+    # We only take the first available php installation, taking the oldest one to be conservative
+	for php_module_name in $(synopkg list --name | grep PHP | sort)
+	do
+        echo "${php_module_name}"
+        return 0
+	done
+	return 0
+}
+
+function guess_php_configuration_file ()
+{
+    local php_bin="$*"
+    local web_station_configuration_file="/var/packages/WebStation/etc/${php_bin}/php.ini"
```

Comment:
From what I gathered this configuration file path is only valid for DSM 6.0+ using WebStation.
To be honest, I have no idea where the php configuration files are stored in older versions... If anyone has a clue...

Reply:
I would not bother with older versions any longer